core/profile.cpp

```diff
@@ -123,7 +122,7 @@ static int get_local_sac(struct plot_info &pi, int idx1, int idx2, struct dive *
 		return 0;
 
 	/* Mean pressure in ATM */
-	depth = (entry1.depth + entry2.depth) / 2;
+	depth_t depth = entry1.depth + entry2.depth / 2;
```

Comment:
This may work out to be correct based on how operator overloading impacts operator precedence, but I think retaining the braces will make this easier to understand.

Reply:
Ouch! Operator overloading can thankfully never change operator precedence. Fixed.